tsconfig.app.json

```diff
@@ -4,7 +4,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "./out-tsc/app",
-    "types": []
+    "types": ["node"]
```

Comment:
Why do you need this?

Reply:
it allows me to use NodeJS types.
i use it so i can use 'NodeJS.Timeout' instead of 'any' in the DragDirective.
should i use 'any' instead?